metadata-ingestion/tests/integration/lookml/lkml_samples_hive/liquid.view.lkml

```diff
@@ -7,7 +7,11 @@ sql:
       FROM
         order
       WHERE
-        {% condition order_region %} order.region {% endcondition %}
```

Comment:
why was this changed?

Reply:
python-liquid template was raising error for `condition` as it is not liquid tag, it is specific to looker and later as it was present in sql, the parser was raising parsing error for this. 

I updated this section to use liquid conditional logic, see the updated file [lkml_samples_hive/liquid.view.lkml](https://github.com/sid-acryl/datahub-fork/blob/master%2Bing-510-lookml-cll/metadata-ingestion/tests/integration/lookml/lkml_samples_hive/liquid.view.lkml)

Reply:
if this is valid lkml, then we need to figure out how to parse it. we can't just change the test cases, since this will show up in people's lkml files.

maybe the solution is replacing the condition tag and everything within it with `TRUE`

Reply:
I added a custom tag implementation which behave similar to looker condition tag. 

Please check the file [looker_liquid_tag.py](https://github.com/datahub-project/datahub/pull/10542#diff-275971787e687c49efe0786b713b7a137feaebe6d8df12c8a924844b3cc3cf23)